src/components/user-select.js

```diff
@@ -0,0 +1,76 @@
+import { Select } from 'antd';
+import PropTypes from 'prop-types';
+import uniqueId from '../utils/unique-id.js';
+import { ensureIsUnique } from '../utils/array-utils.js';
+import { otherUserShape } from '../ui/default-prop-types.js';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+
+const MIN_SEARCH_TEXT_LENGTH = 3;
+
+const filterOption = (inputValue, option) => option.label.toLowerCase().includes(inputValue.toLowerCase());
+
+const createUserLabel = user => user.email ? `${user.displayName} (${user.email})` : user.displayName;
+
+function UserSelect({ value, onChange, onSuggestionsNeeded, ...rest }) {
```

Comment:
Super useful selector!!
We might easily include it in the room members invitation dialog, if we still have time on the project that is..

Reply:
I thought so too, but unfortunately this exposes the user collection which we didn't want to do for non-maintainers, so maybe we have to discuss security issues there.